moaijs/moai
# Chore(lib): Update dependencies

This is just the routine weekly dependency update. Some notes:
- We intentionally upgrade storybook to 6.3, even it's still in beta, to have storybookjs/storybook#14805 and storybookjs/storybook#14850
- Can not update TypeScript to 4.3 due to a conflict. See #203

<hr />

How to test:
- Check the "docs" deployment
- Specifically, check the "Button" page, where the first story ("Basic") should have its source code expanded by default
